src/xpk/commands/batch.py

```diff
@@ -34,10 +34,11 @@ def batch(args: Namespace) -> None:
   Returns:
     None
   """
-  add_zone_and_project(args)
-  set_cluster_command_code = set_cluster_command(args)
-  if set_cluster_command_code != 0:
-    xpk_exit(set_cluster_command_code)
+  if not args.local_test:
```

Comment:
Can we also add it to other commands?

Reply:
I think adding it to other commands requires more changes in the base code. So I'd suggest keeping this PR only as the introduction of Kind and then, as a follow up we can add it to other commands.

Reply:
Yeah, but for job I think we can add.

Reply:
On hold due to #265

Reply:
--kind-cluster flag added to `job` command.